### PR TITLE
Feature/12er grid

### PR DESCRIPTION
**Pull request by tdesero**

### CODE REVIEW

**Review by ASchmidt1024 — APPROVED** [review]
🎸  LGTM.

File: src/es/components/organisms/grid/12er-/12er-.css.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/* css */ `
+:host > section {
+  grid-template-columns: repeat(12, 1fr);
+  gap: ${this.getAttribute('gap') || 'var(--grid-12er-grid-gap, 1rem)'};
```
Comment (Weedshaker) [inline src/es/components/organisms/grid/12er-/12er-.css.js:4]: --grid-12er-grid-gap has one time too many grid. should be --grid-12er-gap

File: src/es/components/organisms/grid/12er-/12er-.css.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    ? ''
+    : `
+      :host > section >*[col-md="${node.getAttribute('col-md')}"] {
+        grid-column: span ${node.getAttribute('col-md') || node.getAttribute('col-sm') || '12'};
+      }
+    `),
+  '')}
+
+  :host > section {
+    gap: ${this.getAttribute('gap-mobile') || this.getAttribute('gap') || 'var(--grid-12er-grid-gap-mobile, var(--grid-12er-grid-gap, 1rem))'};
```
Comment (Weedshaker) [inline src/es/components/organisms/grid/12er-/12er-.css.js:28]: var naming one time too may "grid"

File: src/es/components/organisms/grid/12er-/12er-.css.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  ? ''
+  : `
+    :host > section >*[col-lg="${node.getAttribute('col-lg')}"] {
+      grid-column: span ${node.getAttribute('col-lg') || node.getAttribute('col-md') || '12'};
+    }
+  `),
+'')}
+
+/* col-md */
+@media only screen and (max-width: ${this.getAttribute('tablet-breakpoint') || '1024px'}) {
```
Comment (Weedshaker) [inline src/es/components/organisms/grid/12er-/12er-.css.js:17]: could also use the mobile break point from shadow.js, which covers all common desktop/mobile breakpoints in the project

File: src/es/components/organisms/grid/12er-/12er-.css.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    ? ''
+    : `
+      :host > section >*[col-sm="${node.getAttribute('col-sm')}"] {
+        grid-column: span ${node.getAttribute('col-sm') || node.getAttribute('col-sm') || '12'};
+      }
+    `),
+  '')}
+
+  :host > section {
+    gap: ${this.getAttribute('gap-mobile') || this.getAttribute('gap') || 'var(--grid-12er-grid-gap-mobile, var(--grid-12er-grid-gap, 1rem))'};
```
Comment (Weedshaker) [inline src/es/components/organisms/grid/12er-/12er-.css.js:44]: naming

**Pull request merged**